core/txpool/bundlepool/config.go

```diff
@@ -0,0 +1,73 @@
+package bundlepool
+
+import (
+	"time"
+
+	"github.com/ethereum/go-ethereum/log"
+)
+
+type Config struct {
+	PriceLimit uint64 // Minimum gas price to enforce for acceptance into the pool
+	PriceBump  uint64 // Minimum price bump percentage to replace an already existing transaction (nonce)
+
+	GlobalSlots     uint64 // Maximum number of bundle slots for all accounts
+	GlobalQueue     uint64 // Maximum number of non-executable bundle slots for all accounts
+	MaxBundleBlocks uint64 // Maximum number of blocks for calculating MinimalBundleGasPrice
+
+	BundleGasPricePercentile      uint8         // Percentile of the recent minimal mev gas price
+	BundleGasPricerExpireTime     time.Duration // Store time duration amount of recent mev gas price
```

Comment:
plz refine the comments to match the words(mev gas price -> bundle gas price)

Reply:
Removed useless configuration, please review again.